linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/routers.js

```diff
@@ -36,12 +36,33 @@ var Routers = (function() {
   var clientRE = /^rt\/(.+)\/dst\/id\/(.*)\/requests$/,
       serverRE = /^rt\/(.+)\/srv\/([^/]+)\/(\d+)\/requests$/;
 
+  var mkColor = function() {
+    var colors = [
+      "224,243,219",
+      "204,235,197",
+      "168,221,181",
+      "123,204,196",
+      "78,179,211",
+      "43,140,190",
+      "8,104,172",
+      "8,64,129"
+    ];
+
+    //returns a color based on summed ascii values
+    return function(label) {
+      var idx = _.sumBy(label.split(""), function(char) { return char.charCodeAt(0); });
+      var color = colors[idx % colors.length];
+      return color;
+    }
+  }();
```

Comment:
Yeahhh this happens for the calculator app
![screen shot 2016-03-16 at 1 39 38 pm](https://cloud.githubusercontent.com/assets/549258/13828642/c32c97bc-eb7f-11e5-80d7-84c67855fbb6.png)